GAE/src/org/akvo/flow/domain/DataUtils.java

```diff
@@ -37,10 +36,9 @@
         if (optionResponse == null || optionResponse.trim() == "") {
             return new String[0];
         }
-
-        List<Map<String, String>> optionNodes = new ArrayList<>();
-        if (optionResponse.startsWith("[")) {
```

Comment:
My intention was to rely on the Jackson validator, which throws an `IOException` if the data is not a JSON string, rather than a `startsWith(...)` check. I was unaware of this being an expensive operation. Perhaps I'm missing something?